Verify.md

```diff
@@ -0,0 +1,81 @@
+# Verify Contracts
+
+For each contract whose address appears in migration.json (and therefore is a "migrated contract") we send the contract's solidity code to etherscan.io which will attempt to "verify" that the given code corresponds to the bytecodes deployed at the address given in migration.json.
+
+**Note** Contract validation via this script currently does not work on Windows (truffle-flattener generates invalid flattened .sol files).  The fix for that is at this writing in PR at truffle-flattener.
+
+## Setup
+1. In the project root, create an ".env" file (or use the existing one) that contains:
+```
+APIKEY=[API key from etherscan.io]
+```
+
+Alternately you can supply this as an OS environment variable when you run `build.verify`.
+
+2. initialize just one time, required by `build.verify`:
+
+```script
+npm run verify.initialize
+```
+
+3. build just one time or whenever you modify verify.ts, required by `verify`:
+
+```script
+npm run verify.build
+```
+
+## Verify
+
+### Verify all of the migrated contracts
+
+```script
+npm run verify
```

Comment:
need to know set network ?
Running this produce : `'network' is required.  Run 'npm run help' for cli information. `

Reply:
done